canvas_modules/common-canvas/__tests__/common-properties/form/form-test.js

```diff
@@ -147,7 +148,7 @@ describe("Correct form should be created", () => {
 		expect(isEqual(JSON.parse(JSON.stringify(conditionResource.expectedResult)), JSON.parse(JSON.stringify(generatedForm)))).to.be.true;
 	});
 
-	it("should create a form with editStyle set to subpanel and checkbox panel", () => {
+	it.only("should create a form with editStyle set to subpanel and checkbox panel", () => {
```

Comment:
This will need to be removed.

Reply:
Thanks! Removed, also added icons and labels to 3 more paramDef examples

![image](https://user-images.githubusercontent.com/10677021/102526739-fceec580-4050-11eb-9be6-1bda6ade09cc.png)
![image](https://user-images.githubusercontent.com/10677021/102527061-71c1ff80-4051-11eb-8d26-0d0e43547a69.png)
![image](https://user-images.githubusercontent.com/10677021/102527213-9e761700-4051-11eb-9e5a-0387278f4689.png)